runtime/src/test/java/com/flipkart/flux/resource/StateMachineResourceTest.java

```diff
@@ -92,15 +100,30 @@
     public void setUp() throws Exception {
         objectMapper = new ObjectMapper();
         dbClearRule.explicitClearTables();
+    }
 
+    @BeforeClass
+    public static void beforeClass() {
+        try {
```

Comment:
If the same setup is used multiple places, extract to helper method and use that.